src/main/java/hudson/plugins/jacoco/JacocoPublisher.java

```diff
@@ -302,7 +400,18 @@ protected static String resolveFilePaths(AbstractBuild<?, ?> build, BuildListene
     @SuppressWarnings("resource")
     @Override
     public boolean perform(AbstractBuild<?,?> build, Launcher launcher, BuildListener listener) throws InterruptedException, IOException {
```

Comment:
You need not retain this overload—it is implemented in the superclass to delegate to the new one.